src/cmd/pfs/main.go

```diff
@@ -120,6 +121,14 @@ func main() {
 		},
 	}
 
+	mountCmd := &cobra.Command{
+		Use:  "mount repository-name mount-point",
+		Long: "Mount a repository as a local file system.",
+		Run: func(cmd *cobra.Command, args []string) {
+			check(fuse.Mount(apiClient, args[0], args[1]))
```

Comment:
Add arg length checking for all of the commands. I checked to see of cobra can do this for us and it doesn't seem like it can which is a bit lame :(